Automattic/wp-calypso
# Untangling: Apply new card design in Settings -> Web server

Related to p9Jlb4-eSF-p2#comment-14125

## Proposed Changes

This PR applies the new card design in Settings -> Web server.

> [!NOTE]  
> Further UX refinements (such as submit button handling) are out of scope for now.

|Settings -> Web server|Server Settings|
|-|-|
|<img width="715" alt="image" src="https://github.com/user-attachments/assets/56dfa7d0-80a5-433e-9e8d-c7f6eae32c17">|<img width="567" alt="image" src="https://github.com/user-attachments/assets/55329fb0-027c-450a-b4ee-3721280d4006"><br><br><img width="563" alt="image" src="https://github.com/user-attachments/assets/2c8fc987-d79f-4aa2-8dd1-164629f7de1b">|


## Why are these changes being made?

pbxlJb-6ye-p2

## Testing Instructions

1. Go to `/sites`.
2. Click an Atomic site (preferrably a staging site for more fields in the form).
3. Go to Settings -> Web server subtab.
4. Verify that the functionality still works.
5. Go to `/sites?flags=-untangling/hosting-menu`.
6. Click an Atomic site.
7. Go to Server Settings tab.
4. Verify that the Web server settings and Defensive mode cards still work.

## Pre-merge Checklist
